### ISSUE TITLE
Error when running "Translate" #1 cell in docs/source/examples/Library.ipynb

### ISSUE THREAD

**Issue by DerXter** [OpenNMT/OpenNMT-py#2125]
Hello,

When I run the cell that comes just after the ["Translate" section](https://github.com/OpenNMT/OpenNMT-py/blob/master/docs/source/examples/Library.ipynb), I get a `Keyerror` indicating that the *"features"* field is missing. By adding it as follows, the rest of the code runs fine:
```
src_data = {"reader": onmt.inputters.str2reader["text"](),
            "data": src_val,
            "features": {}}
tgt_data = {"reader": onmt.inputters.str2reader["text"](), 
            "data": tgt_val,
            "features": {}}`

_readers, _data = onmt.inputters.Dataset.config(
    [('src', src_data), ('tgt', tgt_data)])
```
> I put an empty dictionary by referring to the following line `src_nfeats, tgt_nfeats = 0, 0 # do not support word features for now` in the __"Build Fields"__ section. I don't know if this justifies it !?

**Comment by francoishernandez** [comment]
Duplicate of #2110 that should have been fixed by #2112. Are you up to date?

**Comment by DerXter** [comment]
Oh sorry, I didn't see it. Thanks for your help!
